master/reference/public-cloud/aws.md

```diff
@@ -39,34 +39,97 @@ allows Calico to route traffic within a single VPC subnet without using an overl
 
 If you need to split your deployment across multiple AZs for high availability then each AZ will have its own VPC subnet.  To
 use Calico across multiple different VPC subnets or [peered VPCs](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-peering.html),
-in addition to disabling src/dst checks as described above you must also enable ipip encapsulation and outgoing NAT
+in addition to disabling src/dst checks as described above you must also enable IPIP encapsulation and outgoing NAT
 on your Calico IP pools.
 
 See the [IP pool configuration reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
 for information on how to configure Calico IP pools.
 
-By default, Calico's ipip encapsulation applies to all container-to-container traffic.  However,
-encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better performance, you can
-configure Calico to perform ipip encapsulation only across VPC subnet boundaries. To do this, you must
-modify Calico's BIRD config.
+By default, Calico's IPIP encapsulation applies to all container-to-container traffic.  However,
+encapsulation is only required for container traffic that crosses a VPC subnet boundary.  For better 
+performance, you can configure Calico to perform IPIP encapsulation only across VPC subnet boundaries.  
 
-Modify bird_ipam.cfg.template [found here](https://github.com/projectcalico/calicoctl/blob/master/calico_node/filesystem/etc/calico/confd/templates/bird_ipam.cfg.template).
+To do this:
+-  Ensure your node resources are correctly configured with the IPv4 address *and*
```

Comment:
I'm guessing that if we need to say "Ensure X...." then we should be pointing people at the info they need if it's not totally obvious to a bunch of people how to do X.